src/components/generic-elements/alert-component/alert-component.js

```diff
@@ -0,0 +1,30 @@
+import './alert-component.scss'
+import React from 'react'
+import Alert from 'react-bootstrap/Alert'
```

Comment:
Was there a reason we chose react-bootstrap for this? Did this come from Big Jake? We're pretty much standardized on Material UI right now, so I would strongly recommend using that instead of adding another framework: https://material-ui.com/components/dialogs/#alerts

Reply:
per feedback we've removed bootstrap and moved to material UI